### PR TITLE
Alycrase

### PR DESCRIPTION
**Pull request by alycrase**
I've made changes to text ahead of the rebrand from InboxReady to Optimize. Please take a look and make sure everything looks good. I didn't change content for code samples, only the text within.

### CODE REVIEW

File: source/api-email-validation.rst [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 
-.. note:: Our Email Validation service has been renamed to Email Verification service. While the names are different, nothing within our codebase has changed to cause a disruption in service.
+.. note:: Our Single and Bulk Email Validation service is available via the EU API.
 
-.. note:: Our Single and Bulk Email Verification service is available via the EU API.
-
-This API endpoint is an email address verification service.
+This API endpoint is an email address validation service.
 
 We will verify the given address based on:
```
Comment (jeffturpin1987) [inline source/api-email-validation.rst:10]: Should this be "validate"?

File: source/api-email-validation.rst [hunk 2]
```diff
@@ -21,30 +19,30 @@ We will verify the given address based on:
...
+.. note:: The validation feature is rate limited to a set number of active requests at a time. If you receive a 429 error, please wait and try again.
 
 .. code-block:: url
 
      GET /v4/address/validate
 
-Given an arbitrary address, verifies address based off defined checks.
+Given an arbitrary address, validates address based off defined checks.
 
 .. container:: ptable
```
Comment (jeffturpin1987) [inline source/api-email-validation.rst:48]: Should this be "validate"?

File: source/api-email-validation.rst [hunk 3]
```diff
@@ -184,12 +182,12 @@ unknown                          The validity of the recipient address cannot be
...
 
-Bulk Verification
+Bulk Validation
 ~~~~~~~~~~~~~~~~~
 
 .. note::
-    Bulk Verification allows for the verification of a list of email addresses. Given a list name and an uploaded file of email addresses,
-    a backend processing job will be run to verify the list. Once the verifications have all been completed, the
+    Bulk Validation allows for the validation of a list of email addresses. Given a list name and an uploaded file of email addresses,
+    a backend processing job will be run to verify the list. Once the validations have all been completed, the
```
Comment (jeffturpin1987) [inline source/api-email-validation.rst:190]: Should this be "validate"?

File: source/api-email-validation.rst [hunk 4]
```diff
@@ -144,7 +142,7 @@ address                  string       Email address being verified
 did_you_mean             string       (Optional) Null if nothing, however if a potential typo is made to the domain, the closest suggestion is provided
```
Comment (jeffturpin1987) [inline source/api-email-validation.rst:142]: Should the above address parameter have "validated"?

File: source/api-email-validation.rst [hunk 5]
```diff
@@ -277,21 +275,21 @@ Field Explanation:
 Parameter                Type           Description
 =====================    ===========    ============================================================================================================
 created_at               string         Date/Time that the request was initiated
-download_url             array          `csv` and `json` representation of the download link for the results of the bulk verifications
+download_url             array          `csv` and `json` representation of the download link for the results of the bulk validation
 id                       string         list_id name given when the list was initially created
 quantity                 integer        number of total items in the list to be verified
```
Comment (jeffturpin1987) [inline source/api-email-validation.rst:280]: Two instances of "verified", should these be "validated"?

File: source/api-deliverability-alerts.rst [hunk 6]
```diff
@@ -3,7 +3,7 @@
 Alerts
 ======
 
-This section describes our RESTful API for alert configuration for InboxReady
+This section describes our RESTful API for alert configuration for Optimize (formerly InboxReady)
 products.
```
Comment (jeffturpin1987) [inline source/api-deliverability-alerts.rst:9]: Should these uses of "verification" become "validation"?

File: source/api-inboxready-intro.rst [hunk 7]
```diff
@@ -3,12 +3,12 @@
...
+This section documents the API endpoints that support features under the `Optimize`_ offering.
 
-The InboxReady deliverability suite was built to be used seemlessly alongside Mailgun's email
-sending platform. As such, Mailgun and InboxReady share the same underlying API infrustructure,
+The Optimize (formally InboxReady) deliverability suite was built to be used seemlessly alongside Mailgun's email
+sending platform. As such, Mailgun and Optimize share the same underlying API infrustructure,
 such as authentification, REST conformity, etc. To understand the basics of the API or to get
 started, see this `introduction`_.
 
 .. _InboxReady: https://www.mailgun.com/products/inbox/inboxready
```
Comment (jeffturpin1987) [inline source/api-inboxready-intro.rst:13]: Line 13 the first link for "Optimize" still seems to point to IBR. A link I have from marketing is this for the Optimize product:  https://www.mailgun.com/products/optimize/

**Comment by jeffturpin1987** [comment]
Non-urgent question:

@zeefarmer the API endpoints, parameters, and responses are still going to contain InboxReady?  If so, are there any plans to change that as some point to Optimize?

**Comment by zeefarmer** [comment]
> Non-urgent question:
> 
> @zeefarmer the API endpoints, parameters, and responses are still going to contain InboxReady? If so, are there any plans to change that as some point to Optimize?

@jeffturpin1987, we will not be changing any of those right now, but down the road.

**Comment by alycrase** [comment]
@jeffturpin1987 regarding your question, I reached out to Peter earlier with that question and he told me that those are staying InboxReady for now. For now, it's just the written text that needed changes.

File: source/api-deliverability-alerts.rst (hunk 6 above)
Comment (jeffturpin1987) [reply source/api-deliverability-alerts.rst:9]: Lines 25-26 to clarify

File: source/api-email-validation.rst (hunk 1 above)
Comment (jeffturpin1987) [reply source/api-email-validation.rst:10]: Line 10 to clarify.

File: source/api-deliverability-alerts.rst (hunk 6 above)
Comment (alycrase) [reply source/api-deliverability-alerts.rst:9]: @jeffturpin1987 everything is now should be validate, validation, validating.

**Review by alycrase — COMMENTED** [review]
I've updated the link to https://www.mailgun.com/products/optimize/

**Review by zeefarmer — APPROVED** [review]

**Pull request merged**